bgpd/bgp_ecommunity.c

```diff
@@ -443,6 +543,34 @@ static const char *ecommunity_gettoken(const char *str,
 	 * OPQR:    Four byte value
 	 *
 	 */
+	/* IPv6 case : look for last ':' */
+	if (*token == ecommunity_token_rt6 ||
+	    *token == ecommunity_token_val6) {
+		char *limit;
+
+		limit = endptr = strrchr(p, ':');
+		if (!endptr)
+			goto error;
+		endptr++;
+		as = strtoul(endptr, &endptr, 10);
+		if (*endptr != '\0' || as == BGP_AS4_MAX)
```

Comment:
could you format this code snippet a bit more readable? vertical whitespaces would be handy between branches or so..

Reply:
added new lines between if code.